viacoin-blockchain-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "dnsLabelPrefix": {
+      "value": "viacoinnode"
+    },
+    "adminUsername": {
+      "value": "azureuser"
+    },
+    "adminPassword": {
+      "value": "GEN-PASSWORD"
+    },
+    "installMethod": {
+      "value": "From_Source"
+    },    
+    "vmSize": {
+      "value": "Standard_D1"
+    }
```

Comment:
It's not really about blockchain templates but overall best practices for the templates so the deployment experience is consistent across tools - not just in the TravisCI.  As a BP - defaulting should be done in the template file, params for the overrides.